corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -34,5 +34,29 @@ hqDefine("scheduling/js/conditional_alert_main", [
         // setup tab
         basicInformation.setRuleTabVisibility();
         $("#conditional-alert-basic-info-panel").koApplyBindings(basicInformation);
+
+
+        var initialFormContent = {};
```

Comment:
This is a stylistic comment, so feel free to ignore, but it'd be more concise to do these object creations each as a single statement, so this would become:
```
var initialFormContent = _.object(_.map($('.main-form').serializeArray(), function (obj) {
    return [obj.name, obj.value];
}));
```

Reply:
Thanks @orangejenny  this is great, though I'm not sure `_` here is `underscore.js` or `lodash`. It seems `_.object` is removed in lodash with `_.fromPairs`.

Reply:
It's underscore. We do have lodash in `package.json` but only a dependency of other libraries.

I was going to say you can check the dependencies in the `hqDefine` call at the top of the file...and then realized it wasn't there. Can you add it? I'm surprised this page doesn't break on staging/prod (it won't break locally because local requirejs config is more relaxed than prod), underscore must make itself available as a global. But we should be in the habit of declaring all dependencies explicitly.

Reply:
done